task-1/index.ts

```diff
@@ -0,0 +1,41 @@
+import express from "express";
+
+import { userApi } from "./api"
+import { User } from "./types";
+
+const PORT = 3000;
+
+const app = express();
+app.use(express.json());
+
+
+const usersStorage: Array<User> = [];
+const userApiRouter = express.Router();
+userApi(userApiRouter, usersStorage);
+
+const apiV1 = express.Router();
+apiV1.use("/user", userApiRouter);
+apiV1.get("/getAutoSuggestUsers", (req, res) => {
+    const {
```

Comment:
`isDeleted` field will be returned here
it's better to create such method for it

Reply:
Done.